controllers/githubissue_controller.go

```diff
@@ -194,3 +201,22 @@ func (r *GithubIssueReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&trainingv1alpha1.GithubIssue{}).
 		Complete(r)
 }
+
+func (r *GithubIssueReconciler) getMatchingTarget(number int64, url, title string) (*gclient.GithubTicket, error) {
```

Comment:
Same here. Perhaps s/number/issueId/

Reply:
I agre on `issueId`. Ideally, there might be some better way to encapsulate this. I already have a GithubIssue, GithubTicket and gclient.githubIssue, so some refactory later on might help.